arch/arm64/src/common/Make.defs

```diff
@@ -119,3 +119,7 @@ endif
 ifeq ($(CONFIG_ARM64_SEMIHOSTING_HOSTFS),y)
   CMN_CSRCS += arm64_hostfs.c
 endif
+
+ifeq ($(CONFIG_ARCH_HAVE_DEBUG),y)
```

Comment:
You should make a config flag something like CONFIG_ARCH_USE_DEBUG (or similar) to give the user an option to leave the big module arm64_hwdebug.c out.

Reply:
to other pr

Reply:
to other pr